content/technical_challenges.tex

```diff
@@ -71,36 +71,136 @@ \subsection{Visual Referee Challenge}
+            \item Static hand signals with two hands.
+            \item Dynamic (motion) hand signals with a one or two hands.
+        \end{enumerate}
+        
+        The intent of this challenge is to choose a \emph{subset} of potential referee calls in SPL games and test ability of a robot to recognise different types of hand signals in preparation for adoption in RoboCup games, rather than compile a complete set of all referee signals.
 
     \subsubsection{Challenge setup}
 
-        One robot of the challenged team is placed in the center circle facing the head referee standing on the T-junction opposite to the GC. The referee wears red gloves. The robot watching the referee and listening for the whistle. The general procedure is as follows:
+        One robot of the challenged team is placed in the center circle facing the head referee, standing upright (stiffened) with both hands by its side.
+        %Two robots (from one team) are placed on the field, with one robot on the centre circle kick-off position  for a receiving team and one on the team's penalty spot. Both robots should be facing the kick-off spot.
+        %(Note that it is possible to run the challenge up to two teams at the same time. The referee should take care to balance the direction of hand signals in this case).
+        \todo{TW: Multiple robots could be used for this challenge, and the angle of the robot to the referee could make the challenge more difficult.}
+        The head referee must stand on the T-junction of the center field line, opposite to the GC. The referee must wear a ``black-and-white'' stripped referee jersey (if available) and must  wear red gloves. The purpose of this clothing is to clearly distinguish the referee and their hands from background people. 
+        
+        The description of this challenge (and hand-signals) is described based on the viewpoint of the head referee. In these descriptions the ``red team'' is defined as playing from left-to-right, and the ``blue team'' as playing from right-to-left. The use of colours for identifying teams is used to give equivalence to the head referee calls during SPL games.
+        
+        The general procedure is as follows:
 
-        \begin{itemize}
-            \item The referee blows the whistle.
-            \item The referee shows his decision for \qty{15}{\second}.
-            \item During that time or in the additional \qty{10}{\second} the robot has time to phrase the referee's decision, e.g., ```Kick-off left team'''. \todo{Or we use a TCP message}
-        \end{itemize}
+        \begin{enumerate}
+            \item The referee blows the whistle once. The referee may use one hand to hold (and blow) the whistle. The referee may instead leave the whistle held in their mouth without their hands.
+            \item The referee selects and indicates a hand-signal. The referee must indicate this signal for \qty{10}{\second}. \todo{reduced this time as 15s is unnecessarily long.}
```

Comment:
Whistling and signaling should be done afterwards. So give a whistle. Wait a second then show the command.

Reply:
Isn't that what's already written? Or do you want to have a explicit wait-1-second?

Reply:
I think we should add the clearly to indicate that whistelling and signalling should be done after each other. From my experience a lot of referees do both at the same time.

Reply:
Ok, though in practice I don't think the explicit 1 second will make much difference, so long as the hand-signal _follows_ the whistle.

Reply:
I agree.